### DISCUSSION TITLE
Champion's Silk Cowl

### DISCUSSION THREAD

**Discussion by SignorSimone** [chromiecraft/chromiecraft#7196] [Suggestions]
I noticed that "Champion's Silk Cowl" should be the tone of light blue like the rest of the set however it appears as "Champion's Dreadweave Cowl", is it possible to fix this? Thanks :D

**Comment by BeefSteaker** [comment]
I believe textures are handled client side. Are you using of the hd packs?

**Reply by SignorSimone** [reply]
No i dont use hd packs, its my problem or it will be fixed?
